### PR TITLE
[DO NOT MERGE] For SdW laptop

### PR DESCRIPTION
**Pull request by bardliao**
#1359 is merged by accident, so recreate a new PR for it.

This is the difference between my branch and thesofproject/integration/soundwire-latest. I will rebase everyday and QA can take commits from here.

### CODE REVIEW

**Comment by RanderWang** [comment]
@bardliao  @plbossart   how about invoking sdw_clear_slave_status in intel_suspend ?

**Comment by plbossart** [comment]
> @bardliao @plbossart how about invoking sdw_clear_slave_status in intel_suspend ?

why would it help? Isn't it better to update the status when we do all the bus reset, etc.?

File: drivers/soundwire/intel.c [hunk 1]
```diff
@@ -1508,6 +1508,9 @@ static int intel_resume(struct device *dev)
 		return ret;
 	}
 
+	/* make sure all Slaves are tagged as UNATTACHED */
+	sdw_clear_slave_status(&sdw->cdns.bus);
```
Comment (plbossart) [inline drivers/soundwire/intel.c]: ah darn it, yes this was missing. Again a copy/paste error, rather not doing a copy/paste....

Thanks @bardliao I will apply this asap to the integration/soundwire-intel branch.

Nice catch!

**Comment by plbossart** [comment]
@bardliao this PR is no longer needed, right? all your changes are in integration/soundwire-latest and I tested that we can get sound without any edits/changes from that branch.

**Comment by bardliao** [comment]
> @bardliao this PR is no longer needed, right? all your changes are in integration/soundwire-latest and I tested that we can get sound without any edits/changes from that branch.

Yeah. Let's close it.

**Pull request closed without merging**